spec/requests/medications_spec.rb

```diff
@@ -153,51 +138,39 @@ def post_create(medication_params)
     let!(:valid_medication_params) { { name: 'Updated Medication Name' } }
 
     def put_update(medication_params)
-      put :update, params: { id: medication.id, medication: medication_params }
+      put medication_path(medication), params: { medication: medication_params }
     end
 
-    context 'when the user is logged in' do
-      include_context :logged_in_user
+    context 'when signed in' do
+      before { sign_in user }
 
       context 'when valid params are supplied' do
+
         it 'updates a medication' do
           put_update valid_medication_params
-          expect(assigns(:medication)[:name]).to eq(valid_medication_params[:name])
+          expect(response.body).to include(valid_medication_params[:name])
```

Comment:
This test currently doesn't pass because the updated medication name appears in the response body reformatted from `Updated Medication Name` to `updated-medication-name` (in redirect url). I assume this is a transformation applied by FriendlyId, but I'm not sure how to mirror that transformation here.

Alternatively, I could either drop this example, or change the redirect example below to something like
```suggestion
          it 'redirects to renamed medication page' do
            let!(renamed_medication) { 
              create(:medication, user_id: user.id, name: valid_medication_params[:name])
            }
            put_update valid_medication_params
            expect(response).to redirect_to(medication_path(renamed_medication))
          end
```

Reply:
You can use [parameterize](https://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-parameterize) to "slugify" the title.

So the following would result:

```ruby
include ActiveSupport::Inflector
parameterize("Updated Medication Name")

=> "updated-medication-name"
```